content/en/docs/collector/internal-telemetry.md

```diff
@@ -0,0 +1,166 @@
+---
+title: Internal telemetry
+weight: 25
+cSpell:ignore: journalctl kube otecol pprof tracez zpages
+---
+
+You can check the health of any Collector instance by checking its own internal
+telemetry. Read on to learn how to use this data to diagnose and troubleshoot
+Collector issues.
+
+## Activate internal telemetry in the Collector
+
+By default, the Collector exposes its own telemetry in two ways:
+
+- Internal [metrics](#configure-internal-metrics) are exposed using a Prometheus
+  interface which defaults to port `8888`.
+- [Logs](#configure-internal-logs) are emitted to `stdout` by default.
+
+[Traces](#configure-internal-traces) are not exposed by default. Two feature
+gates offer experimental support for a configuration based on the OpenTelemetry
+Configuration schema.
+
+### Configure internal metrics
+
+You can configure how internal metrics are generated and exposed by the
+Collector. By default, the Collector generates basic metrics about itself and
+exposes them for scraping at http://127.0.0.1:8888/metrics. You can expose the
+endpoint to one specific or all network interfaces when needed. For
+containerized environments, you might want to expose this port on a public
+interface.
+
+Set the address in the config `service::telemetry::metrics`:
+
+```yaml
+service:
+  telemetry:
+    metrics:
+      address: '0.0.0.0:8888'
+```
+
+To visualize these metrics, you can use the
+[Grafana dashboard](https://grafana.com/grafana/dashboards/15983-opentelemetry-collector/),
+for example.
```

Comment:
```suggestion
```

@open-telemetry/docs-maintainers are we OK with linking out to this dashboard here? My concern is less that it is a vendor website (since we reference grafana oss at a few places already), but if we want to approve this 3rd party dashboard over others.

Reply:
Can we create our own Grafana dashboard and host it here? For example, I can contribute the one from https://github.com/mhausenblas/ref.otel.help/tree/main/otel-collector-ops as the starting point (which was developed based on our experiences)?

Reply:
Removed in 9f2d105. Resolving conversation.